IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/QuantitySets/Qto_SignBaseQuantities/DocQuantitySet.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_SignBaseQuantities" UniqueId="fb485ee8-9c7e-46ca-9d0f-eac5dfd7aa41">
+	<Quantities>
+		<DocQuantity xsi:nil="true" href="Height_3jq0d8r0zBYvccpej6okcm" />
+		<DocQuantity xsi:nil="true" href="Width_2vkF8AsQL5fg4keqWbdE8G" />
+		<DocQuantity xsi:nil="true" href="Thickness_29hrivl4D9Xv3IjkS6FDr" />
```

Comment:
Thickness surprised me here a bit.

Reply:
well the sign plate has a thickness and if its a smart sign they are quite chunky! do you prefer depth?

Reply:
My surprise was more in the line of when would it be a required quantity. Is it something that is required in the chunky signs? For operation, I would not expect this, Maybe in operation?

Reply:
it a BOQ value construction need to know what material to get to make the sign out of eg thickness of steel plate to buy.